### ISSUE TITLE
dapr message invocation with vpn doesn't work after 20 seconds

### ISSUE THREAD

**Issue by mPoelen** [microsoft/azure-container-apps#83]
Hi,

I have setup a local dapr configuration. Installed dapr and the neccesary containers are running in docker. When running 2 different web api's with visual studio the communicate with dapr message invocation works. When switching to a vpn connection this communication works for max 20 secondes after the last message invocation and after that i get a 500 response from the request. Does anybody know a solution or at least at direction where to look at.

Thanks

**Comment by greenie-msft** [comment]
Hi @mPoelen, 

Can you please elaborate on your local environment setup? How are you performing service invocation, is it through a Dapr SDK?

**Comment by mPoelen** [comment]
Yes its trough a dapr SKD in .Net

Here is the class with the method invokation:
![image](https://user-images.githubusercontent.com/85079687/150933686-1683c231-a8ef-41f8-b89c-7fd2eed6b62d.png)

Here i have dapr settings from the appsettings.development.json
![image](https://user-images.githubusercontent.com/85079687/150934017-d2034225-d490-4c2d-b238-ce8ab265dbe5.png)

This is the log console message that appears every 30 seconds without vpn and without doing the invokation.
![image](https://user-images.githubusercontent.com/85079687/150934372-f5738fef-8ae1-4655-9707-ab9cb5b6cb81.png)

This is the log message that appears every 30 seconds with vpn and without doing the invokation:
![image](https://user-images.githubusercontent.com/85079687/150934773-826f1f6f-f55e-44ee-b170-9dab16723700.png)

This is a part of the response when doing a invokation with the time out message
![image](https://user-images.githubusercontent.com/85079687/150934954-0030ddac-81e4-49aa-b0ce-9c86433113b9.png)

Do you need more info?

**Comment by kendallroden** [comment]
What type of machine are you using?

**Comment by mPoelen** [comment]
yes, its about service to service communication. What network settings do i need to change?

**Comment by kendallroden** [comment]
Can you confirm the answers to the question above? What type of machine?

**Comment by kendallroden** [comment]
The answer will most likely depend on the type of machine and the policies used. Are you able to replicate these issues with all dapr svc to svc calls on your machine? This problem would be consistent. On my Mac I go to network settings and have to disable a content filter that runs as part of Microsoft defender for Mac. Not sure if that is applicable to you

**Comment by mPoelen** [comment]
I have a windows machine and i am able to replicate the issue. The issues also popup with other services and other svc to svc calls. Oke i will look into the content filter.  Thanks, if you have more directions let me know. what i get out of the log it seems that the correct ip address is removed from the mDNS cache.

**Comment by kendallroden** [comment]
Were you able to look into this on your side or find any potential running agent that might be blocking proper local resolution?
